app-modules/multifactor-authentication/src/Livewire/MultifactorAuthenticationManagement.php

```diff
@@ -187,7 +187,7 @@ public function getFormSchema(): array
             Toggle::make('is_sso_enabled')
                 ->helperText('Your account is configured using enterprise single sign-on (SSO) so multifactor authentication is handled by
         your institution, rather than by this product.')
-                ->label('SSO Enabled')
+                ->label('Product MFA Disabled')
```

Comment:
This label only needs to be changed if the User logs in via SSO. Which is indicated by the User having `is_external` being `true`.

So I would change this to a function and check if the Users `is_external` attribute and do this new wording if `true` and the old wording if not.